Assets/Plugins/Editor/Uplift/Export/PackageExportData.cs

```diff
@@ -27,16 +27,19 @@
 using System;
 using Object = UnityEngine.Object;
 
-namespace Uplift.Export {
+namespace Uplift.Export
+{
 #if UNITY_5_1_OR_NEWER
     [CreateAssetMenuAttribute(fileName = "PackageExport.asset", menuName = "Uplift/Package Export Definition", order = 250)]
 #endif
-    class PackageExportData : ScriptableObject, ICloneable {
+    class PackageExportData : ScriptableObject, ICloneable
+    {
 
         [Header("Basic Package Information")]
         public  string    packageName     =  "";
         public  string    packageVersion  =  "";
         public  string    license         =  "";
+        public  Object    templateUpset   = new Object();
```

Comment:
any reason we don't call this `templateUpset` I thought it was the template itself first :)

Reply:
nvm, I am done reading code today...